### ISSUE TITLE
Wrong default value is generated for integer parameters

### ISSUE THREAD

**Issue by asicignano** [swagger-api/swagger-core#1472]
The Swagger generated for parameters with a type of integer are surrounded with quotes, when they should not be. This results in errors when attempting to use the Swagger in the editor, the Swagger Import tool for AWS, etc.

Example:

 @QueryParam("page") @DefaultValue("1") @ApiParam(value = "Page to be returned") Integer page
generates

      parameters:
        - name: page
          in: query
          description: 'Page to be returned'
          type: integer
          default: "1"
          format: int32

the default should be without quotes:

      parameters:
        - name: page
          in: query
          description: 'Page to be returned'
          type: integer
          default: 1
          format: int32

**Comment by lugaru1234** [comment]
@asicignano, what swagger version do you use?
The issue does exist in 1.5.3, but it is fixed in 1.5.4-SNAPSHOT
This test works
```
public class DefaultValueTest {

    @Test
    public void scanResourceWithDefaultValue() {
        final Swagger swagger = new Reader(new Swagger()).read(ResourceWithDefaultValue.class);
        QueryParameter qp = (QueryParameter) swagger.getPath("/test").getGet().getParameters().get(0);
        final String json = "{" +
                "   \"name\":\"page\"," +
                "   \"in\":\"query\"," +
                "   \"description\":\"Page to be returned\"," +
                "   \"required\":false," +
                "   \"type\":\"integer\"," +
                "   \"default\":1," +
                "   \"format\":\"int32\"" +
                "}";
        SerializationMatchers.assertEqualsToJson(qp, json);
    }

    @Api("/test")
    @Path("/test")
    public class ResourceWithDefaultValue {
        @ApiOperation(value = "test")
        @GET
        public void getTest(@QueryParam("page") @DefaultValue("1") @ApiParam(value = "Page to be returned") Integer page) {
            return;
        }
    }
}
```

**Comment by webron** [comment]
Closing this as fixed in 1.5.4 (release to come shortly). If that's not the case, please reopen.
